docs/reference/esql/esql-across-clusters.asciidoc

```diff
@@ -19,6 +20,49 @@ include::{es-ref-dir}/search/search-your-data/search-across-clusters.asciidoc[ta
 
 include::{es-ref-dir}/search/search-your-data/search-across-clusters.asciidoc[tag=ccs-proxy-mode]
 
+[discrete]
+[[esql-ccs-security-model]]
+==== Security model
+
+[discrete]
+[[esql-ccs-security-model-api-key]]
+===== API key authentication
+
+[NOTE]
+====
+The `ENRICH` keyword is *not supported* until 8.15 when using {esql} with the API key based security model.
+====
+
+The following information pertains to using {esql} across clusters with the <<remote-clusters-api-key, *API key based security model*>>.
+API key based cross-cluster search (CCS) enables more granular control over allowed actions between clusters. 
+
+You will need to create an API key using the <<security-api-create-cross-cluster-api-key,Create cross-cluster API key>> API or using the {kibana-ref}/api-keys.html[Kibana API keys UI].
+
+The following example API call creates a role that can query remote indices using {esql} with the API key based security model.
+
+[source,console]
+----
+POST /_security/role/remote1
+{
+  "indices": [
+    { 
+      "names" : [""], <2>
+      "privileges": ["read"]
+    }
+  ], 
+  "remote_indices": [
+    {
+      "names": [ "logs-*" ],
```

Comment:
can add a call out here. it's kinda self evident, but I think spelling it out can help. 
for example, 
```
<3> The indices allowed be read in the remote cluster. The configured <<?? how to configure the cross cluster api key ??, cross-cluster API key>> must also allow this index to be read. 
```